packages/mermaid/src/diagrams/class/classDiagram.spec.ts

```diff
@@ -1193,16 +1193,18 @@ describe('given a class diagram with relationships, ', function () {
     });
 
     it('should handle relation definitions with type only on right side', function () {
-      const str = 'classDiagram\n' + 'Class1 --|> Class02';
+      const str = 'classDiagram\n' + 'Class1 --|> Class2';
```

Comment:
We already support Extension `--|>` and Realization `..|>`. 
Wouldn't releasing this change mark the extension as a realization, changing the meaning of a lot of existing diagrams?

```mermaid
classDiagram
classA <|-- classB
classM <|.. classN
```